### PR TITLE
Ot_EventChain1_CreatedWithPOM

### PR DESCRIPTION
**Pull request by olenat2019**
User Story https://trello.com/c/MDGRoJfL/65-entity-events-chain-1
Test Cases https://trello.com/c/h2W0l1Jq/466-entity-events-chain-1

### CODE REVIEW

**Review by ArinaJur — APPROVED** [review]

File: src/test/java/EntityEventsChain1Test.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        mainPage.clickMenuEventsChain1();
+
+        Chain1Page chain1Page = new Chain1Page(getDriver());
+        chain1Page.clickNewFolder()
+                  .inputInitialValue()
+                  .clickSaveButton();
+
+        Assert.assertEquals(chain1Page.getRowCount(),1);
+        Assert.assertEquals(chain1Page.getRow(0),expectedValues);
+
```
Comment (Stan575) [inline src/test/java/EntityEventsChain1Test.java:30]: Unnecessary empty line

File: src/test/java/EntityEventsChain1Test.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    }
+
```
Comment (Stan575) [inline src/test/java/EntityEventsChain1Test.java:32]: Unnecessary empty lines

File: src/test/java/model/BaseViewPage.java [hunk 3]
```diff
@@ -17,6 +17,7 @@ public BaseViewPage(WebDriver driver) {
     }
 
     public List<String> getValues() {
+
```
Comment (Stan575) [inline src/test/java/model/BaseViewPage.java:20]: Unnecessary empty line

File: src/test/java/model/Chain1EditPage.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    @FindBy(id = "f1")
+    WebElement f1;
+
+    @FindBy(id = "f10")
+    WebElement f10;
+
+    @FindBy(id = "pa-entity-form-save-btn")
+    WebElement saveButton;
+
+
```
Comment (Stan575) [inline src/test/java/model/Chain1EditPage.java:20]: Unnecessary empty line, one empty line is enough for separation

File: src/test/java/model/Chain1EditPage.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    public Chain1EditPage(WebDriver driver) {
+        super(driver);
+    }
+    @Override
+    protected Chain1Page createPage() {
+        return new Chain1Page(getDriver());
+    }
+
+
+
```
Comment (Stan575) [inline src/test/java/model/Chain1EditPage.java:30]: Unnecessary empty lines

File: src/test/java/model/Chain1EditPage.java [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    public Chain1EditPage inputInitialValue() {
+        ProjectUtils.sendKeys(f1, "1");
+        getWait().until(ExpectedConditions.attributeToBeNotEmpty(f10, "value"));
+        return this;
+    }
+
+
```
Comment (Stan575) [inline src/test/java/model/Chain1EditPage.java:37]: Unnecessary empty lines

File: src/test/java/model/Chain1Page.java [hunk 7]
```diff
@@ -0,0 +1,19 @@
+package model;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+
```
Comment (Stan575) [inline src/test/java/model/Chain1Page.java:6]: Unnecessary empty line

File: src/test/java/model/Chain1Page.java [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+public class Chain1Page extends BaseTablePage<Chain1Page,Chain1EditPage> {
+
+    public Chain1Page(WebDriver driver) {
+        super(driver);
+
```
Comment (Stan575) [inline src/test/java/model/Chain1Page.java:11]: Unnecessary empty line

File: src/test/java/model/Chain1ViewPage.java [hunk 9]
```diff
@@ -0,0 +1,10 @@
+package model;
+
+import org.openqa.selenium.WebDriver;
+
+public class Chain1ViewPage extends BaseViewPage{
+
+    public Chain1ViewPage(WebDriver driver){ super(driver); }
+
+
```
Comment (Stan575) [inline src/test/java/model/Chain1ViewPage.java:9]: Unnecessary empty line

File: src/test/java/model/MainPage.java [hunk 10]
```diff
@@ -32,6 +32,10 @@
     @FindBy(css = "#menu-list-parent>ul>li>a[href*='id=62")
     private WebElement menuEventsChain2;
 
+    @FindBy(css = "#menu-list-parent>ul>li>a[href*='id=61']")
+    private WebElement menuEventsChain1;
+
+
```
Comment (Stan575) [inline src/test/java/model/MainPage.java:38]: Unnecessary empty line

File: src/test/java/model/Chain1ViewPage.java [hunk 11]
```diff
@@ -0,0 +1,10 @@
+package model;
+
+import org.openqa.selenium.WebDriver;
+
+public class Chain1ViewPage extends BaseViewPage{
+
+    public Chain1ViewPage(WebDriver driver){ super(driver); }
```
Comment (Stan575) [inline src/test/java/model/Chain1ViewPage.java:7]: constructor method in one line, method body should be in the next line

**Review by Stan575 — COMMENTED** [review]
Overall looks great
Would be nice to have style consistent

**Review by PeterKuzyk — APPROVED** [review]

**Pull request merged**